src/activities/content/ContentScormActivityEntity.js

```diff
@@ -26,16 +25,17 @@ export class ContentScormActivityEntity extends ContentEntity {
 	}
 
 	/**
-	 * @returns {Date|undefined} The date and time the scorm activity was last edited
+	 * @returns {Date|undefined} The date and time the scorm activity was last edited according to the content service
 	 */
-	lastModified() {
-		const lastModifiedSubEntity = ContentHelperFunctions.getLastModifiedSubEntity(this._entity);
-
-		if (!lastModifiedSubEntity) {
-			return null;
-		}
+	contentServiceLastModified() {
+		return this._entity && this._entity.properties && this._entity.properties.contentServiceScormActivityLastModified;
+	}
 
-		return lastModifiedSubEntity.properties.date;
+	/**
+	 * @returns {string|undefined} Name of the Scorm actvity according to the content service
+	 */
+	contentServiceName() {
```

Comment:
nit: change this to `contentServiceTitle` instead of name (this will also require lms changes in https://github.com/Brightspace/lms/pull/14130)